lab_2_keywords_cooccurrence/main.py

```diff
@@ -40,7 +103,13 @@ def calculate_frequencies_for_content_words(candidate_keyword_phrases: KeyPhrase
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not check_sequence(candidate_keyword_phrases, tuple, False):
+        return None
+    keyword_freq = {}
+    for phrase in candidate_keyword_phrases:
+        for word in phrase:
+            keyword_freq.update({word: (1 if word not in keyword_freq else keyword_freq.get(word, 0) + 1)})
```

Comment:
why use update instead of ` keyword_freq[word] = ...` ?

Reply:
also, using `get` let's you avoid all the `if... else` business

Reply:
done